app/models/link.rb

```diff
@@ -1,4 +1,6 @@
 class Link < ActiveRecord::Base
+  after_update :reset_time_and_status, if: :url_changed?
```

Comment:
Thanks for writing up that story @sihugh, we've come up against this problem a few times during this story. We didn't attempt to fix it in this story, as there's a few considerations to uh consider about how to best model the URL related fields for a LocalAuthority (see comment above in description)